packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -50,7 +50,7 @@ describe('Approve token for deposit', () => {
        * If confirm spending fails, test is still considered to be passing by Cypress
        * We add another check to make sure the test fails if needed
        */
-      cy.wait(10_000)
+      cy.wait(25_000)
```

Comment:
Orbit chains may take longer, sometimes the tests fail because they don't have enough time to run

Reply:
can we only make it `25_000` when it's running for an Orbit chain?

Reply:
I think it's good to have it for both, I'm only guessing it's for Orbit only as I've observed in the videos. But to be sure I'd leave it for both.

Reply:
Anyway, 15 extra seconds doesn't matter because we always wait for withdrawal e2e to finish at the end, they take a bit longer